engine/schema/src/main/resources/META-INF/db/schema-420to421.sql

```diff
@@ -242,3 +242,6 @@ INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Advanced', 'DEFAULT', 'manag
 INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Advanced', 'DEFAULT', 'management-server', 'vmware.hung.wokervm.timeout', '7200', 'Worker VM timeout in seconds');
 INSERT IGNORE INTO `cloud`.`configuration` VALUES ("Alert", 'DEFAULT', 'management-server', "alert.smtp.connectiontimeout", "30000", "Socket connection timeout value in milliseconds. -1 for infinite timeout.");
 INSERT IGNORE INTO `cloud`.`configuration` VALUES ("Alert", 'DEFAULT', 'management-server', "alert.smtp.timeout", "30000", "Socket I/O timeout value in milliseconds. -1 for infinite timeout.");
+
+-- Update the value of user.vm.readonly.details record in the configuration table to "" if it is NULL
+UPDATE `cloud`.`configuration` SET value = '' WHERE name = 'user.vm.readonly.details' AND value IS NULL;
```

Comment:
I believe that this is not the correct script for this update :smile:

Reply:
@Pearl1594 I think we want this line in the (to be created) 4.19.2 -> 4.19.3 and in the 4.20.0 -> 4.20.1 scripts. We must also test if this upgrade will then work from 4.19.3 to 4.20.1 (it should according to theory but testing is still advised (by me))

code looks good btw